src/video/color.cpp

```diff
@@ -34,22 +34,25 @@ Color::Color() :
   alpha(1.0f)
 {}
```

Comment:
You'll want to initialize the ultra_violet member here as well, otherwise the builds will fail.